mne_bids/read.py

```diff
@@ -29,42 +30,41 @@
           '.set': io.read_raw_eeglab}
 
 
-def _read_raw(raw_fpath, electrode=None, hsp=None, hpi=None, config=None,
-              montage=None, verbose=None, allow_maxshield=False):
+def _read_raw(raw_fpath, allow_maxshield, verbose=None):
     """Read a raw file into MNE, making inferences based on extension."""
-    _, ext = _parse_ext(raw_fpath)
+    fname, ext = _parse_ext(raw_fpath)
+    bids_root, bids_fname = op.split(fname)
 
     # KIT systems
     if ext in ['.con', '.sqd']:
-        raw = io.read_raw_kit(raw_fpath, elp=electrode, hsp=hsp,
-                              mrk=hpi, preload=False)
+        mrk = _find_matching_sidecar(bids_fname, bids_root,
+                                     '_markers.???')
+        raw = io.read_raw_kit(raw_fpath, elp=None, hsp=None,
```

Comment:
`mne-bids` only copies the `mrk` file but `mne-python` expects also `elp` and `hsp` to be provided. This tells me that the writing of this file is already broken. cc @sappelhoff